frontend/src/pages/GenExamPage.js

```diff
@@ -94,65 +241,221 @@ const GenExamPage = () => {
+				<Grid item xs = {4}>
+					<Typography className={classes.upperSideText} variant="h5" >Personaliza o teu exame</Typography>
+				</Grid> 
+				<Grid item xs = {1}>
+					<Typography className={classes.ou} variant="h6">ou</Typography> {/*Maybe put this in Bold */}
+				</Grid>
+				<Grid item xs = {4}>
+					<Typography className={classes.upperSideText} variant="h5">Deixa isso connosco</Typography>
+				</Grid>
 			</Grid>
-			<Grid item>
-				
+			}
+			<Grid direction={is1100pxScreen ? "column" : "row"} className={classes.lowerPanel} container xs = {11}>
+
+				<Grid justifyContent= {is1100pxScreen ? "center" : "flex-start"} container xs ={is1100pxScreen ? 10 :  5}> {/*left lower panel*/}
+					<Grid justifyContent= {is1100pxScreen ? "center" : "flex-start"} alignContent="flex-start" container xs = {6}> {/*Pick Subject*/}
+						<Grid item>
+							<Typography variant = "h6"> 1 - Disciplina </Typography>
+						</Grid>
+						<Grid justifyContent= {is1100pxScreen ? "center" : "flex-start"} container>
+							<Select
+							 IconComponent={ArrowDropDownRoundedIcon}
+							 onChange={handleChangeSubject}
+							 id="grouped-select"
+							 defaultValue={subject}
+							 classes={{ root: classes.selectRoot }}
+							 className={classes.select} 
+							 MenuProps = {{classes:{paper:classes.paper}}}
+							 inputProps = {{classes:{icon:classes.icon}}}
+							 disableUnderline>
+								<MenuItem  classes={{ selected: classes.selected, root: classes.rootMenuItem }}  value="none"> <em>Nenhuma</em></MenuItem>
+								<ListSubheader className={classes.subheader}> <Typography>Ciências e Tecnologias</Typography></ListSubheader>
+									<MenuItem classes={{ selected: classes.selected, root: classes.rootMenuItem }} value={"math"}> <Typography variant = "h6">Matemática A</Typography></MenuItem>
+									<MenuItem classes={{ selected: classes.selected, root: classes.rootMenuItem }} value={"physics"}> <Typography variant = "h6">Física e Química</Typography></MenuItem>
+								
+								<ListSubheader className={classes.subheader}> <Typography>Línguas e Humanidades</Typography></ListSubheader>
+									<MenuItem classes={{ selected: classes.selected, root: classes.rootMenuItem }}> <Typography variant = "h6">História A</Typography></MenuItem>
+							</Select>
+						</Grid>
+					</Grid> 
+
+					<Grid justifyContent="center" container xs = {is1100pxScreen ? 12 : 6}> {/* Pick year*/}
+						<Grid item>
+							<Typography variant = "h6"> 2 - Ano(s) </Typography>
+						</Grid>	
+						<Grid justifyContent= {is1100pxScreen ? "center" : "center"} container>
+							<FormControl className={classes.boxes}>
+								<FormGroup >
+									<FormControlLabel labelPlacement="start" control={<Checkbox checked={options.randomGrade} icon = {<GreyRoundCheckbox/>} checkedIcon = {<RedRoundCheckmark/>} onChange={handleChangeRandomGrade} name="randomGrade"/>} label={<Typography variant = "h6">Aleatório</Typography>}/>
```

Comment:
Maybe create a separate wrapper component for the `Checkbox` as it can be reused throughout the platform.

Reply:
Good idea!